00_responsive_web_design/00_tribute_page/style.css

```diff
@@ -1,32 +1,35 @@
-body {
-    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
-    max-width: 500px;
-    padding: 1em;
-  }
-  
-  #title {
-    text-align: center;
-  }
-  
-  #img-div {
-    max-width: 300px;
-    margin: auto;
-    font-size: 0.666em;
-  }
-  /*
-    horizontally centered image
-  */
-  #img-div img#image {
-    display: block;
-    margin: auto;
-    max-width: 100%;
-    height: auto;
-    margin-bottom: 5px;
-  }
-  #img-div figcaption#img-caption {
-    text-align: center;
-  }
-  
-  #tribute-info {
-    margin-top: 33px;
-  }
\ No newline at end of file
+body#main {
+  font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+  max-width: 500px;
+  padding: 1em;
+  margin: auto;
+}
+
+#title {
```

Comment:
As an example of PR comments, please add the HTML element to these IDs. So this would be `h1#title`. This isn't something I would normally care about, but just want to test out the automation. Thanks :)